packages/jest-util/src/create_process_object.js

```diff
@@ -90,6 +90,7 @@ export default function() {
   }
 
   newProcess.env = createProcessEnv();
+  newProcess.send = () => {};
```

Comment:
Any other functions on it while we're here?

Reply:
I've thought about it and I don't think so. Maybe `newProcess.exit`, but I thougth it was better to let it exit.

Reply:
Might be cool to do a "`process.exit()` called from test" or something. Rarely on purpose, is it?